cdn-proto/schema/messages_capnp.rs

```diff
@@ -586,6 +586,7 @@ pub mod message {
 pub enum Topic {
   Global = 0,
   Da = 1,
+  Transactions = 2,
 }
 
 impl ::capnp::introspect::Introspect for Topic {
```

Comment:
I think instead we want to update `messages.capnp` and then run something like `capnp compile -o rust cdn-proto/schema/*.capnp`